### ISSUE TITLE
No result from wifi.recon / Could not detect gateway

### ISSUE THREAD

**Issue by Benji-Collins** [bettercap/bettercap#567]
I am trying to view all networks around me so I can then perform a deauth attack on a specific network. When running wifi.recon on, I don't seem to find any networks. Running wifi.show doesn't help much either. Below I've shown what I try to run. When I run with debugging, I see something about the gateway not being found? I don't know heaps about how Bettercap works. I am using my bluetooth connection for ssh (as wifi is supposed to be being used for monitoring) but I don't see why that would interfere. Running `iwconfig` confirms that the device is in monitor mode. I am happy to provide more info if it helps anyone. Thanks.

### Environment

Please provide:

* Bettercap version: bettercap v2.24 (built for linux arm with go1.12.4)
* OS version: Raspbian Stretch (Debian 9) on a Raspberry Pi Zero W, with Nexmon firmware patch to enable monitor mode.
* Go version: 1.12.4 arm
* Full debug:
![Debug](https://i.imgur.com/dkLWzbP.png)

### Steps to Reproduce

1. Run `bettercap -iface wlan0`
2. Run `wifi.recon on`

**Expected behavior:** 
I expected to be presented with a list of wifi networks near me.

**Actual behavior:**
Nothing seems to happen.

**Comment by evilsocket** [comment]
Did you try to `monstart` first? Check https://github.com/seemoo-lab/nexmon/issues/45

**Comment by Benji-Collins** [comment]
Yes I did. I also tried a fresh install and I couldn't get it to work either. I then installed the aircrack-ng suite and ran `airmon-ng start wlan0` then `bettercap -iface wlan0mon -eval "wifi.recon on"` and managed to pick up wifi networks at expected. After a few minutes though bettercap stopped working properly. `airodump-ng wlan0mon` and `aireplay-ng -0` worked fine for a deauth attack though, so I will probably just stick with them.
